Command/InitAclCommand.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\AclBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Security\Acl\Dbal\Schema;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\SchemaException;
+
+/**
+ * Installs the tables required by the ACL system.
+ *
+ * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * @final since version 3.4
+ */
+class InitAclCommand extends Command
+{
+    protected static $defaultName = 'acl:init';
+
+    private $connection;
+    private $schema;
+
+    public function __construct(Connection $connection, Schema $schema)
+    {
+        parent::__construct();
+
+        $this->connection = $connection;
+        $this->schema = $schema;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Mounts ACL tables in the database')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command mounts ACL tables in the database.
+
+  <info>php %command.full_name%</info>
+
+The name of the DBAL connection must be configured in your <info>app/config/security.yml</info> configuration file in the <info>security.acl.connection</info> variable.
```

Comment:
`in your <info>app/config/security.yml</info> configuration file` <- it's not compatible with modern Symfony apps (it should be `config/packages/security.yaml`). What if you reword the message to be more neutral?

```
The name of the DBAL connection must be configured in the <info>security.acl.connection</info>
variable of your main security configuration file.
```

Reply:
Changed it to this, because it seems the config node was wrong in the example:
```
The name of the DBAL connection must be configured in the 
<info>acl.connection</info> variable of your acl configuration file.
```